docs/fundamentals/code-analysis/quality-rules/ca2251.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "CA2251: Use `String.Equals` over `String.Compare`"
+description: "Learn about code analysis rule CA2251: Use `String.Equals` over `String.Compare`"
+ms.date: 05/31/2021
+ms.topic: reference
+f1_keywords:
+- CA2251
+- UseStringEqualsOverStringCompare
+helpviewer_keywords:
+- UseStringEqualsOverStringCompare
+- CA2251
+author: NewellClark
+dev_langs:
+- CSharp
+- VB
```

Comment:
When there is no code snippets in the article, the `dev_langs` is better to be removed.